vr-xcon/xcon.act

```diff
@@ -1,8 +1,34 @@
 import net
 import time
+import file
 
-actor TcpEndpoint(connect_auth, dns, name, interface):
+actor Healthcheck(env, expected):
+    # keep track of TcpEndpoint actor states (ids, like "r1/1")
+    var states = {}
+
+    fa = file.FileAuth(env.auth)
+    wfa = file.WriteFileAuth(fa)
```

Comment:
This part was nicer in your first program where you built the wfa in the main actor. Now, since you've sent `env` to the Healthcheck actor, it has access to do anything, like shut down the system. It's generally an anti-pattern sending `env` around. Best practice is to send the most precise auth needed...

Reply:
I agree. This was just one of the things I moved around while trying to get it to compile with a dictionary of actors.